node/overseer/src/lib.rs

```diff
@@ -989,7 +1001,7 @@ impl<M> OverseenSubsystem<M> {
 	/// Send a message to the wrapped subsystem.
 	///
 	/// If the inner `instance` is `None`, nothing is happening.
-	async fn send_message(&mut self, msg: M) -> SubsystemResult<()> {
+	async fn send_message(&mut self, msg: M, origin: &'static str) -> SubsystemResult<()> {
```

Comment:
Should we add a secondary ` fn send_message_anon(&mut self, msg: M)` to avoid the clutter through the source?

Reply:
the anon would only be used in tests in one file, I'm not sure it's worth it

Reply:
Your call, just thought it's a small addition for some easy IDE aided lookups, your call :)